src/bosh_azure_cpi/spec/manual_tests/performance/run_performance.sh

```diff
@@ -0,0 +1,85 @@
+chruby 2.4.4
+source ./run_performance.pri.sh
+
+export SERVICE_HOST_BASE="core.windows.net"
+
+pushd tf
+terraform init
+
+export RESOURCE_GROUP=${PREFIX}-rg
+export VNET_NAME=${PREFIX}-vnet
+export SUBNET_NAME=${PREFIX}-bosh-subnet
+default_storage_account=$(echo $PREFIX | sed -e 's/[^0-9a-z]//g')
```

Comment:
Seems that $PREFIX should be converted into lowercase.

Reply:
good suggestion.